modules/virt-downloading-virtctl-client-offline.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// virt/install/virt-installing-virtctl.adoc
+
+[id="virt-installing-virtctl-client-offline_{context}"]
+= Downloading the virtctl client offline
+
+Download the `virtctl` client without an internet connection using the client provided in a pod in the cluster.
+
+.Prerequisites
+
+* You must have an activated {product-title} subscription to access the download page on the Customer Portal.
+
+.Procedure
+
+. View the `ConsoleCLIDownload` object by running the following command:
++
+[source,terminal]
+----
+$ oc get ConsoleCLIDownload virtctl-clidownloads-kubevirt-hyperconverged -o yaml
+----
+
+. Download the `virtctl` client using the link listed for your distribution. All downloads are in `tar.gz` format.
```

Comment:
All downloads are not in "tar.gz" format.  Linux artifact is in "tar.gz" whereas Mac+Windows are in "zip".  This is in this way for `oc` client as well and we did just mimic it. 

```
  links:
  - href: https://hyperconverged-cluster-cli-download-openshift-cnv.apps.verify-49.cnv-qe.rhcloud.com/amd64/linux/virtctl.tar.gz
    text: Download virtctl for Linux for x86_64
  - href: https://hyperconverged-cluster-cli-download-openshift-cnv.apps.verify-49.cnv-qe.rhcloud.com/amd64/mac/virtctl.zip
    text: Download virtctl for Mac for x86_64
  - href: https://hyperconverged-cluster-cli-download-openshift-cnv.apps.verify-49.cnv-qe.rhcloud.com/amd64/windows/virtctl.zip
    text: Download virtctl for Windows for x86_64

```